### ISSUE TITLE
Separate Objects

### ISSUE THREAD

**Issue by Tom2Drake** [helpsterTee/Unity-IFCEngine#1]
First of all, thanks for the absolut amazing work. Is there any way to sperate an object like a window? So that afterwards we have a frame and the window glas als separate objects?

**Comment by berry4u** [comment]
Thanks for sharing your work. Have you continued this work (I presume related to your PhD) ?
Is there some way to share your wrapping project. I'm involved in BIM research too and it would be very useful form me.
BN

**Comment by helpsterTee** [comment]
@tom2drake: only if it's defined like that in the IFC file. I'm currently assigning the material to the whole IfcWindow. It would be possible with IfcBuildingProxyElements. 

@berry4u:
The library uses IfcEngine which is commercial. I have to rework this to XBim in the near future. So this project is not dead, I'm just occupied with my new job at the moment ;)

**Comment by DevPaulVarghese** [comment]
Hi, thank you for your amazing work. I just wanted to know what the status is of the issues and tasks that are yet to be completed?

**Comment by helpsterTee** [comment]
Well, XBim and Unity together seems to be a trainwreck. I've tested and it had several issues.

Unity Reflect should bring support for Autodesk Revit:  https://blogs.unity3d.com/2019/06/05/unity-reflect-bim-to-real-time-3d-in-one-click-for-better-design-decisions/

For IFC files, I'll write a webservice parser. That way XBim's .NET stuff and unity's won't get confused by each other. 

Most likely something along the lines of https://docs.microsoft.com/en-us/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api

**Comment by DevPaulVarghese** [comment]
Aah, so will you be implementing XBim library for the Unity-IFCEngine Project such that you can create a web-application of the build using a .NET application correct? Or am I slight mistaken?

**Comment by helpsterTee** [comment]
Nah, it will be two applications. XBim .NET with a webservice and Unity-IFCEngine calling the webservice for geometry and metadata.
